changelog.txt

```diff
@@ -1,123 +1,314 @@
-0.4.1
-
+=========
+Changelog
+=========
+
+The **garnett** package follows `semantic versioning <https://semver.org/>`_.
+
+Version 0.5
+===========
+
+Highlights
+----------
+	- Remove acceleration as supported property.
+	- Remove the read_gsd_shape_data flag from GSD reader.
+	- Raise AttributeError if accessing an invalid property.
+	- Add GETAR file reader/writer.
+	- Add support for various particle properties
+	- Extend reader/writer shape support
+	- Add automatic format detection to reader/writer
+	
```

Comment:
I don't think we need a highlights section. Same applies below multiple times

Reply:
Yeah, I think the changelog is mostly self-explanatory. To reduce the changelog redundancy and to reduce the possible work of writing highlights in the future, I agree they can be removed.

Reply:
In  a3dd8ff